pkg/kubectl/cmd/set/set_subject.go

```diff
@@ -262,7 +262,9 @@ func (o *SubjectOptions) Run(fn updateSubjects) error {
 			continue
 		}
 
-		return o.PrintObj(actual, o.Out)
+		if err := o.PrintObj(actual, o.Out); err != nil {
```

Comment:
same return error exists above in the DryRun block

Reply:
@liggitt I fix this in #64242